pallets/subtensor/src/lib.rs

```diff
@@ -1656,13 +1665,44 @@ pub mod pallet {
         pub fn get_priority_set_weights(hotkey: &T::AccountId, netuid: u16) -> u64 {
             if Uids::<T>::contains_key(netuid, &hotkey) {
                 let uid = Self::get_uid_for_net_and_hotkey(netuid, &hotkey.clone()).unwrap();
+                let stake = Self::get_total_stake_for_hotkey(&hotkey);
+                if stake <= 20_000_000_000_000_000 { return 0; } // Blacklist weights transactions for low stake peers.
```

Comment:
Should this limit be a hyperparameter? Also is the limit suppose to be 20 million tao?